app/javascript/src/components/header-center.vue

```diff
@@ -1,19 +1,14 @@
 <template>
   <div class="header-center">
     <div class="header-center__title">
-      Poder recomendador de ideas para <span class="header-center__title--emphasis-word"> {{receiverName}} </span>
+      Poder recomendador de ideas para <span class="header-center__title--emphasis-word"> {{receiverName}}</span>
```

Comment:
Expected 1 space after '{{', but not found   vue/mustache-interpolation-spacing<br>Expected 1 space before '}}', but not found  vue/mustache-interpolation-spacing